Cargo.toml

```diff
@@ -4,10 +4,15 @@ version = "0.1.0"
 authors = ["Victor Berger <victor.berger@thalesgroup.com>"]
 
 [dependencies]
-wayland-server = "0.8.4"
+wayland-server = { version = "0.8.4", features = ["dlopen"] }
```

Comment:
Update to 0.8.6 for fixing the missing symbol error.

Also, we should make the "dlopen" feature conditioned on the glutin feature of smithay